website/content/terminal/guides/intros/stocks/index.md

```diff
@@ -1,225 +1,364 @@
+┏━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┓
+┃                ┃ SPY          ┃
+┡━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━┩
+│ Name           │ SPDR S&P 500 │
+├────────────────┼──────────────┤
+│ Price          │ 407.69       │
+├────────────────┼──────────────┤
+│ Open           │ 408.77       │
+├────────────────┼──────────────┤
+│ High           │ 410.00       │
+├────────────────┼──────────────┤
+│ Low            │ 404.75       │
+├────────────────┼──────────────┤
+│ Previous Close │ 407.68       │
+├────────────────┼──────────────┤
+│ Volume         │ 52,458,993   │
+├────────────────┼──────────────┤
+│ 52 Week High   │ 479.98       │
+├────────────────┼──────────────┤
+│ 52 Week Low    │ 348.11       │
+├────────────────┼──────────────┤
+│ Change         │ 0.01         │
+├────────────────┼──────────────┤
+│ Change %       │ 0.00%        │
+└────────────────┴──────────────┘
+```
 
-|           | Name                               | Country       | Sector     | Industry                  | Exchange       |
-| --------- | ---------------------------------- | ------------- | ---------- | ------------------------- | -------------- |
-| MSF.BR    | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Belgium        |
-| MSF.DE    | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Germany        |
-| MSF.F     | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Germany        |
-| MSF2.L    | Leverage Shares 2x Microsoft ETC A | None          | None       | None                      | United-Kingdom |
-| MSF3.L    | Leverage Shares 3x Microsoft ETC   | None          | None       | None                      | United-Kingdom |
-| MSFS.L    | Leverage Shares -1x Microsoft ETC  | None          | None       | None                      | United-Kingdom |
-| MSFT.BA   | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Argentina      |
-| MSFT.MX   | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Mexico         |
-| MSFT      | Microsoft Corporation              | United States | Technology | Software - Infrastructure | USA            |
-| MSFT.SN   | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Chile          |
-| MSFT.VI   | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Austria        |
-| MSFT34.SA | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Brazil         |
-| SMSF.L    | Leverage Shares -1x Microsoft ETC  | None          | None       | None                      | United-Kingdom |
+### Codes
+
+`codes` prints the CIK, Composite FIGI, Share Class FIGI, and SIC codes - when available - for a US-listed stock. This function requires a free API key from Polygon.
+
+```console
+load aapl/codes
```

Comment:
Let's place this in a separate code block so people can copy.

Reply:
done